src/main/java/ch/njol/skript/expressions/ExprLastDeathLocation.java

```diff
@@ -0,0 +1,78 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import org.bukkit.Location;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Last Death Location")
+@Description("The location of a player's last death. Location is saved over server restarts. The last death location can only be set for online players.")
+@Examples("set {lastdeathlocation::%player's uuid%} to last death location of player")
```

Comment:
I think this example isn't really a good example as the syntax itself saves over a restart. The main example I could think of was some sorta hint to their last death as this will probably be the most used case.
```suggestion
@Examples({
	"on respawn:",
		"\twhile last death location is set:",
			"\t\twait a second",
			"\t\tsend action bar \"&cLast Death Location %last death location%\" to player",
			"\t\tdistance between player and last death location is less than 5",
			"\t\tmessage \"You found your death location!\"",
			"\t\tclear last death location of player",
			"\t\tstop",
	"",
	"command /stopDeathNotification:",
		"\ttrigger:",
			"\t\tclear last death location of player",
			"\t\tmessage \"Stopping last death actionbar notification\""
})
```
This example works, but I didn't test with this syntax installed, so testing may be plausible.

Reply:
Unresolved